devconf-outreachy/source/index.rst

```diff
@@ -0,0 +1,126 @@
+
+.. Becoming OpenStack Developer via Outreachy slides file, created by
+   hieroglyph-quickstart on Wed Jan  2 14:19:51 2019.
+
+
+Becoming OpenStack Developer via Outreachy
+==========================================
+
+Presented by: Aija Jauntēva, Dmitry Tantsur, Richard Pioso, Ilya Etingof
+
+What is OpenStack
+-----------------
+
+* Decade-old open source community project
+* Has involves XXX developers from YYY companies
+* Made of XXX repos totalling YYY commits
+
+What is Outreachy
+-----------------
+
+* Outreachy breeds new open source engineers
+* Outreachy fights the discrimination
+* Interns get on-board open source projects
+* Each intern gets a mentor (or two)
+* 3 month remote paid internship
+
+Why enrolling into the internship
+---------------------------------
+
+* To find your aspirations
+* To learn the software engineering craft
+* To get along with the open source trade guild
+* To get a job (hopefully, the dream one)
+
+Why mentoring with @Outreachy
+-----------------------------
+
+* Sustain the open source culture
+* OpenStack helps breeding open source engineers
+* Diversity is healthy, discrimination is unfair
+
+Aija's Internship @OpenStack
+----------------------------
+
+* How it all begun
+* The initial application:
+
+  - Challenges
+  - Successes
+
+Outreachy application period
+----------------------------
+
+In the eyes of a mentor:
+
+* Come up with isolated, yet non-trivial and useful tasks
+* On-board a handful of candidates to let them proceed
+* Pondering code reviews aiming to merge their work
+
+OpenStack: everybody struggles
+------------------------------
+
+* Technology
+
+  - Code complexity and its sheer size is overwhelming
+  - The developer's workbench takes time to learn
+  - Not always possible to run OpenStack at home
+
+* Social
+
+  - Code reviews tend to be slow
+  - Sometimes feedback can look harsh
+  - Newcomers are prone to taking the feedback personally
+  - It takes time become a good community citizen
+
+Aija's survival hints
+---------------------
+
+...
+
+Case study: reaching out for the industry
```

Comment:
Compared to other slides, this slide contains a lot more text. Maybe this can be split in several slides or the text can be made more concise?

Reply:
Absolutely! I'm not fixing this only because I know your PR is on the way which might address this. ;-)

Reply:
Unfortunately my PR did not address this as I worked on gaps, not updating existing content.
We need to decide who is talking about this slide and then take it from there.

Reply:
I am thinking to ask @rpioso if he'd be up for revealing the industry's view on the collaboration, open source and internship...?

Reply:
It just occurred to me that it is in the tradition of devconf for the track chairperson to introduce the speaker(s) to the audience.